packages/components/inputs/async-select-input/src/async-select-input.js

```diff
@@ -197,7 +202,10 @@ AsyncSelectInput.propTypes = {
   onBlur: PropTypes.func,
   onChange: PropTypes.func.isRequired,
   onFocus: PropTypes.func,
+  openMenuOnFocus: PropTypes.bool,
```

Comment:
Maybe we can define a shared naming here as also defaulting. We often have disable* which I somewhat subjectivity like more than enable*. It can still default to true. Then all these behavior flags could be disableOpenMenuOnClick etc.